tests/v2/validation/provisioning/hosted/README.md

```diff
@@ -0,0 +1,204 @@
+
+
+
+## Hosted Provider Provisioning Configs
+
+For your config, you will need everything in the Prerequisites section on the previous readme along with and at least one [Cloud Credential](#cloud-credentials) and [Hosted Provider Config](#hosted-provider-configs). 
+
+Your GO test_package should be set to `provisioning/hosted`.
+Please see below for more details for your config. 
+
+1. [Prerequisites](../README.md)
+2. [Cloud Credential](#cloud-credentials)
+3. [Hosted Provider Config](#hosted-provider-configs)
+4. [Back to general provisioning](../README.md)
+
+Below are example configs needed for the different hosted providers including GKE, AKS, and EKS. In order to run these tests, the [cloud credentials](#cloud-credentials) are also needed. GKE (googleCredentials), AKS(azureCredentials), and EKS(awsCredentials)
+
+## Cloud Credentials
+
+### AWS
+```json
+"awsCredentials": {
+   "secretKey": "",
+   "accessKey": "",
+   "defaultRegion": ""
+  },
+```
+### Azure
+```json
+"azureCredentials": {
+   "clientId": "",
+   "clientSecret": "",
+     "subscriptionId": "",
+     "environment": "AzurePublicCloud"
+  },
+```
+### Google
+```json
+"googleCredentials": {
+    "authEncodedJson": ""
+}
+```
+
+## Hosted Provider Configs
+
+### EKS Cluster Config
+```json
+"eksClusterConfig": {
+  "imported": false,
+  "kmsKey": "",
+  "kubernetesVersion": "1.21",
+  "loggingTypes": [],
+  "nodeGroups": [
+    {
+      "desiredSize": 2,
+      "diskSize": 20,
+      "ec2SshKey": "",
+      "gpu": false,
+      "imageId": "",
+      "instanceType": "t3.medium",
+      "labels": {},
+      "maxSize": 2,
+      "minSize": 2,
+      "nodegroupName": "",
+      "requestSpotInstances": false,
+      "resourceTags": {},
+      "subnets": [],
+      "tags": {}
+
+    }
+  ],
+  "privateAccess": true,
+  "publicAccess": true,
+  "publicAccessSources": [],
+  "region": "us-east-2",
+  "secretsEncryption": false,
+  "securityGroups": [""],
+  "serviceRole": "",
+  "subnets": ["", ""],
+  "tags": {}
+},
+```
+### AKS Cluster Config
+```json
+"aksClusterConfig": {
+  "dnsPrefix": "-dns",
+  "imported": false,
+  "kubernetesVersion": "1.21.9",
+  "linuxAdminUsername": "azureuser",
+  "loadBalancerSku": "Standard",
+  "networkPlugin": "kubenet",
+  "nodePools": [
+    {
+      "availabilityZones": ["1", "2", "3"],
+      "nodeCount": 1,
+      "enableAutoScaling": false,
+      "maxPods": 110,
+      "mode": "System",
+      "name": "agentpool",
+      "osDiskSizeGB": 128,
+      "osDiskType": "Managed",
+      "osType": "Linux",
+      "vmSize": "Standard_DS2_v2"
+    }
+  ],
+  "privateCluster": false,
+  "resourceGroup": "",
+  "resourceLocation": "eastus",
+  "tags": {}
+},
+```
+### GKE Cluster Config
+note that the following are required and should be updated:
```

Comment:
We should have it be `Note that the following are required and should be updated:`